src/javascript/JContent/actions/openInJcontentAction.jsx

```diff
@@ -31,7 +32,8 @@ export const OpenInJContentActionComponent = ({path, render: Render, loading: Lo
                 isVisible={res.checksResult}
                 enabled={res.checksResult}
                 onClick={() => {
-                    const acc = registry.find({type: 'accordionItem', target: 'jcontent'}).find(acc => acc.canDisplayItem(res.node));
+                    const params = getCanDisplayItemParams(res.node);
```

Comment:
and here `={folderNode:res.node}`
```suggestion
                    const params = {folderNode: res.node};
```